diagnostic.md

```diff
@@ -38,7 +40,11 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+$base-margin2:($base-margin/2);
+$base-margin: 1.5em;
+li {
+margin: $base-margin2;
+}
 ```
```

Comment:
Correct execution, you'd just want to define `$base-margin2` after `$base-margin`, otherwise `$base-margin2` cannot access the variable `$base-margin`.